tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -422,7 +423,19 @@ func addSchedulerCommandFunc(cmd *cobra.Command, args []string) {
 // NewScatterRangeSchedulerCommand returns a command to add a scatter-range-scheduler.
 func NewScatterRangeSchedulerCommand() *cobra.Command {
 	c := &cobra.Command{
-		Use:   "scatter-range [--format=raw|encode|hex] <start_key> <end_key> <range_name>",
+		Use:        "scatter-range [--format=raw|encode|hex] <start_key> <end_key> <range_name>",
+		Short:      "scatter-range will be deprecated in the future, please use scatter-range-scheduler instead",
+		Run:        addSchedulerForScatterRangeCommandFunc,
```

Comment:
Can we remove `short` and `run` directly?

Reply:
Generally, the deprecation process is gradual, so I'm still keeping it and then removing it completely after a period of time (the exact time is still up for discussion)

Reply:
Got it, make sense to `run`.
When `Deprecated` is added, the command will be considered `unavailable`. This means this command will not appear in `help`, so maybe we can remove `short` directly?

Reply:
Oh, I don't know that. I will do a test for it and choose a best practice.

Reply:
`Short` has been removed. Thanks for your suggestion~